.github/workflows/inactive_issue_checker.yml

```diff
@@ -0,0 +1,39 @@
+name: Check Inactive Issues
+permissions: read-all
```

Comment:
These should be {} (I know it's not consistent yet for the different workflows). Individual steps should specify additional permissions if needed. Please fix, thanks.

Reply:
Change this to `permissions: {}`. See previous comment.

If individual steps below need permissions, specify them in the step.